solution/build.sh

```diff
@@ -0,0 +1,183 @@
+readarray -t DEP_IMAGES < deps.txt
+
+function clean()
+{
+	rm -rf ${BUILD_ROOT}
+}
+
+function mkdirs()
+{
+	mkdir -p ${ISO_ROOT}
+}
+
+function gen_manifest_yaml()
+{
+	cat > ${ISO_ROOT}/manifest.yaml <<EOF
+apiVersion: solutions.metalk8s.scality.com/v1alpha1
+kind: SolutionConfig
+metadata:
+  annotations:
+    solutions.metalk8s.scality.com/display-name: ${PRODUCT_NAME}
+    solutions.metalk8s.scality.com/git: ${GIT_REVISION}
+    solutions.metalk8s.scality.com/development-release: true
+    solutions.metalk8s.scality.com/build-timestamp: ${BUILD_TIMESTAMP}
+    solutions.metalk8s.scality.com/build-host: ${BUILD_HOST}
+  name: ${PRODUCT_LOWERNAME}
+spec:
+  version: ${VERSION_FULL}
+  images:
+    - zenko-operator:${OPERATOR_TAG}
+  operator:
+    image:
+      name: zenko-operator
+      tag: ${OPERATOR_TAG}
+EOF
+}
+
+function copy_yamls()
+{
+	cp -R -f operator/ ${ISO_ROOT}/operator
+	sed "s/VERSION_FULL/${VERSION_FULL}/" zenko-version-cr.yaml > ${ISO_ROOT}/operator/zenko-version-cr.yaml
+}
+
+function download_dependency_yamls()
+{
+	OPERATOR_DIR=${ISO_ROOT}/operator
+	CRDS_DIR=${OPERATOR_DIR}/crds
+
+	CERT_VERSION=$(grep /cert-manager-controller: deps.txt | awk -F ':' '{print $2}')
+	wget -O ${CRDS_DIR}/cert_manager_crd.yaml https://github.com/jetstack/cert-manager/releases/download/${CERT_VERSION}/cert-manager.yaml
+
+	# combine all zookeeper files into a bundle.yaml
+	ZOOKEEPER_VERSION=$(grep /zookeeper-operator: deps.txt | awk -F ':' '{print $2}')
+	ZOOKEEPER_CRD_FILE=${CRDS_DIR}/zookeeper_operator_crd.yaml
+	wget -O ${BUILD_ROOT}/zookeeper-operator.tar.gz https://github.com/pravega/zookeeper-operator/archive/v${ZOOKEEPER_VERSION}.tar.gz
+	mkdir ${BUILD_ROOT}/zookeeper-operator
+	tar -C ${BUILD_ROOT}/zookeeper-operator --strip-components=1 -xf ${BUILD_ROOT}/zookeeper-operator.tar.gz
+	cp ${BUILD_ROOT}/zookeeper-operator/deploy/crds/zookeeper_v1beta1_zookeepercluster_crd.yaml ${ZOOKEEPER_CRD_FILE}
+	echo --- >> ${ZOOKEEPER_CRD_FILE}
+	cat ${BUILD_ROOT}/zookeeper-operator/deploy/all_ns/rbac.yaml >> ${ZOOKEEPER_CRD_FILE}
+	echo --- >> ${ZOOKEEPER_CRD_FILE}
+	cat ${BUILD_ROOT}/zookeeper-operator/deploy/all_ns/operator.yaml >> ${ZOOKEEPER_CRD_FILE}
+}
+
+
+function gen_operator_yaml()
+{
+	# we need to escape / with \/ in our sed command
+	sed "s/REPLACE_IMAGE/zenko-operator:${OPERATOR_TAG}/" operator.yaml > ${ISO_ROOT}/operator/operator.yaml
+}
+
+function gen_operator_config_yaml()
+{
+	CONFIG_PATH=${ISO_ROOT}/operator/operator-config.yaml
+
+	echo apiVersion: solutions.metalk8s.scality.com/v1alpha1 > ${CONFIG_PATH}
+	echo kind: OperatorConfig >> ${CONFIG_PATH}
+	echo repositories: >> ${CONFIG_PATH}
+  	echo "  ${VERSION_FULL}:" >> ${CONFIG_PATH}
+    	echo "    - endpoint: metalk8s-registry/${PRODUCT_LOWERNAME}-${VERSION_FULL}" >> ${CONFIG_PATH}
+      	echo "    images:" >> ${CONFIG_PATH}
+	for dep in ${DEP_IMAGES[@]}; do
+		SHORT_DEP=${dep##*/}
+        	echo "      - ${SHORT_DEP}" >> ${CONFIG_PATH}
+	done
+}
```

Comment:
We don't need to generate this file, it will be done by MetalK8s and passed to the Operator using the `--config` option.
Plus, the generated YAML is wrong (`images` key is under indented):
```suggestion
function gen_operator_config_yaml()
{
	CONFIG_PATH=${ISO_ROOT}/operator/operator-config.yaml

	echo apiVersion: solutions.metalk8s.scality.com/v1alpha1 > ${CONFIG_PATH}
	echo kind: OperatorConfig >> ${CONFIG_PATH}
	echo repositories: >> ${CONFIG_PATH}
  	echo "  ${VERSION_FULL}:" >> ${CONFIG_PATH}
    	echo "    - endpoint: metalk8s-registry/${PRODUCT_LOWERNAME}-${VERSION_FULL}" >> ${CONFIG_PATH}
      	echo "      images:" >> ${CONFIG_PATH}
	for dep in ${DEP_IMAGES[@]}; do
		SHORT_DEP=${dep##*/}
        	echo "        - ${SHORT_DEP}" >> ${CONFIG_PATH}
	done
}
```

Reply:
Actually, it makes a lot of sense to build such a config file for use in online context (but probably just build it as part of the Operator image). This would ease testing outside of MetalK8s.

Reply:
Oh wait... this file is passed INTO the operator, I thought it was the operator handling it and passing it to metal.

Reply:
Taking this change for now and can adjust later